.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,13 +1,27 @@
-<!--  Delete any sections below that are not relevant to this PR -->
+<!-- Insert GitHub issue number below -->
+Fixes #
 
 ## Description
-<!--- What are the changes? -->
+<!-- Briefly describe the proposed changes -->
 
-## Context
-<!--- Why did you make these changes, and why in this particular way? -->
+## Notes
+<!-- Be sure to mention anything unusual, out-of-scope or new technical debt, etc -->
 
-## References
-<!-- Include links to JIRA, Github, etc. if appropriate -->
+## Checklist
+<!-- Acknowledge the general checklist below -->
+- [ ] I verify the build is in a non-broken state
+- [ ] I verify all changes are within scope of the linked issue
+
+<!-- Acknowledge which type of PR this is -->
+- [ ] This PR contains CSS changes
+- [ ] This PR does not contain CSS changes
+
+<!-- Acknowledge or delete all steps below based on type of PR -->
+- [ ] I re-generated all CSS files under dist folder
```

Comment:
This is the same as the verify the build is in a non-broken state, maybe we can combine those?

Reply:
I think it's slightly different. The build could be broken due to lint error,  babel, gulp, and a bunch of other things.

Reply:
Agreed. But in general is `npm run build` either generates files or fails that should be an invalid usecase. Which we can say: `npm run build` is run and no other files are generated.

Reply:
Yep, definitely an argument here that it might be redundant, however if I have them together it breaks my "branching" approach ;-) I personally think another reminder in the "CSS" type section doesn't hurt.

Reply:
Sure, I'm fine with it. Let leave it.